packages/core/src/browser/new-element-contribution.ts

```diff
@@ -0,0 +1,153 @@
+/** ******************************************************************************
+ * Copyright (c) 2023 CrossBreeze.
+ ********************************************************************************/
+import { Command, CommandContribution, CommandRegistry, MaybePromise, MenuContribution, MenuModelRegistry, URI, nls } from '@theia/core';
+import { CommonMenus, DialogError, codicon, open } from '@theia/core/lib/browser';
+import { TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { injectable } from '@theia/core/shared/inversify';
+import { FileStat } from '@theia/filesystem/lib/common/files';
+import { FileNavigatorContribution, NavigatorContextMenu } from '@theia/navigator/lib/browser/navigator-contribution';
+import { WorkspaceCommandContribution } from '@theia/workspace/lib/browser/workspace-commands';
+import { WorkspaceInputDialog } from '@theia/workspace/lib/browser/workspace-input-dialog';
+
+const NEW_ELEMENT_NAV_MENU = [...NavigatorContextMenu.NAVIGATION, '0_new'];
+const NEW_ELEMENT_MAIN_MENU = [...CommonMenus.FILE, '0_new'];
+
+interface NewElementTemplate extends Command {
+    label: string;
+    fileExtension: string;
+    content: string | ((name: string) => string);
+}
+
+const NEW_ELEMENT_TEMPLATES: NewElementTemplate[] = [
```

Comment:
@martin-fleck-at 

I see you add Entity, Relationship and Diagram in this order to this collection, but it doesn't appear in this order everywhere.

For the 'folder'-menu it is correct, but for the context and file menu it's not adhering the order of this array. Can you adjust it so the order is adhered?

Reply:
Yes, I fixed the order in which the items are specified. Good catch, thank you!